### PR TITLE
add many jade/top providers from addon Monazite

### PR DESCRIPTION
**Pull request by DancingSnow0517**
## What
add many jade/top providers

- show the machine parallel
- show the working machine recipe outputs
- show the multiblock is valid or not
- show the multiblock maintenance info, like the requiered tool and so on
- show steam machine exhaust vent info, like dirction and is blocked
- show the machin auto output dirction and is allow input from output side
- show the cable voltage and amperage
- show the machine current mode when machin have more than 1 recipe type
- show the block color when it stained

## Additional Information
image
image
image
image
image
image
image
image

### CODE REVIEW

File: src/generated/resources/assets/gtceu/lang/en_us.json [hunk 1]
```diff
@@ -3701,6 +3714,8 @@
   "gtceu.top.progress_computation": " / %s CWU",
   "gtceu.top.progress_sec": " / %s s",
   "gtceu.top.progress_tick": " / %s t",
+  "gtceu.top.recipe_output": "Recipe Outputs:",
+  "gtceu.top.stained": "Stained: %s",
```
Comment (screret) [inline src/generated/resources/assets/gtceu/lang/en_us.json]: IMO this tooltip should be either "Dyed: %s" or "Colored: %s"

File: src/generated/resources/assets/gtceu/lang/en_us.json [hunk 2]
```diff
@@ -1922,7 +1917,6 @@
   "compass.node.gtceu.multiblock/multiblock_tank": "Multiblock Tank",
   "compass.node.gtceu.multiblock/primitive_blast_furnace": "Primitive Blast Furnace",
   "compass.node.gtceu.multiblock/primitive_pump": "Primitive Pump",
-  "compass.node.gtceu.multiblock/processing_array": "Processing Array",
```
Comment (screret) [inline src/generated/resources/assets/gtceu/lang/en_us.json]: You need to enable the`doProcessingArray` and `generateLowQualityGems` configs.

**Review by screret — CHANGES_REQUESTED** [review]

File: src/generated/resources/assets/gtceu/lang/en_us.json (hunk 1 above)
Comment (DancingSnow0517) [reply src/generated/resources/assets/gtceu/lang/en_us.json]: renamed

File: src/generated/resources/assets/gtceu/lang/en_us.json (hunk 2 above)
Comment (DancingSnow0517) [reply src/generated/resources/assets/gtceu/lang/en_us.json]: fixed

**Review by screret — APPROVED** [review]

**Pull request merged**